build/configs/rtl8730e/flat_dev/defconfig

```diff
@@ -286,18 +286,21 @@ CONFIG_BOARDCTL_RESET=y
 # Board-Specific Options
 #
 CONFIG_FLASH_START_ADDR=0x8000000
-CONFIG_FLASH_SIZE=16777216
+CONFIG_FLASH_SIZE=33554432
 CONFIG_FLASH_VSTART=0x8000000
+# CONFIG_BOARD_MEMORY_8M is not set
+CONFIG_BOARD_MEMORY_64M=y
```

Comment:
The defconfig of loadable_ext does not have these configs. And I can't find Kconfig for this. Could you check?